slider.js

```diff
@@ -1,3 +1,13 @@
+(function(factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['angular', 'bootstrapSlider'], factory);
```

Comment:
Why 'bootstrapSlider' and not 'bootstrap-slider' like in the original bootstrap-slider project ???